Directory.Build.props

```diff
@@ -477,6 +477,19 @@
     </ProjectReference>
   </ItemDefinitionGroup>
 
+  <ItemGroup>
```

Comment:
Why are these being defined in the root props file?

Reply:
Because they are now used by testPackages.proj and by sendtohelix.proj, didn't want to duplicate them so I defined them in a shared location.

Reply:
Per our discussion let's move these into the packageTest.targets then only specify PackageOutputPath from the helix command.